test/check-machines-settings

```diff
@@ -644,10 +644,15 @@ class TestMachinesSettings(VirtualMachinesCase):
         # Check rebooting machine will not unexpectedly affect watchdog configuration
         setWatchdogActionLive(action="poweroff", reboot_machine=True)
 
-        # Older libvirt in rhel-8-9 and centos-8-stream doesn't support target.hotplug=off
-        if not m.image.startswith("rhel-8") and m.image != "centos-8-stream":
+        m.execute("virsh destroy subVmTest1")
+
+        # Some OSes don't support target.hotplug=off
+        if m.image.startswith("rhel-8") or m.image in ["centos-8-stream", "debian-stable"]:
+            self.assertIn("Unknown --controller options", m.execute(
+                "! virt-xml subVmTest1 --edit model=pci-root --controller target.hotplug=off 2>&1")
+            )
+        else:
```

Comment:
> So I guess the better solution is to assert virt-xml --version < 4.0.0 . Or maybe assert both libvirt with virsh --version < 7.9.0 and virt-xml since we can't guarantee which package update will make it sooner to the debian-stable?

Or this approach. That should also be fine